<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DB_project</class>
 <widget class="QMainWindow" name="DB_project">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>940</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1024</width>
    <height>940</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1024</width>
    <height>940</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>DB_project</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="login_group">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>241</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="id_lab">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>32</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Blippo Blk BT</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>ID</string>
     </property>
    </widget>
    <widget class="QLabel" name="pw_lab">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>66</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Blippo Blk BT</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>PW</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="id_txt">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>27</y>
       <width>101</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>나눔바른고딕</family>
      </font>
     </property>
     <property name="cursor" stdset="0">
      <cursorShape>IBeamCursor</cursorShape>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhNone</set>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="tabChangesFocus">
      <bool>true</bool>
     </property>
     <property name="lineWrapMode">
      <enum>QTextEdit::NoWrap</enum>
     </property>
    </widget>
    <widget class="QTextEdit" name="pw_txt">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>61</y>
       <width>101</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>나눔바른고딕</family>
      </font>
     </property>
     <property name="cursor" stdset="0">
      <cursorShape>IBeamCursor</cursorShape>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhNone</set>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="tabChangesFocus">
      <bool>true</bool>
     </property>
     <property name="lineWrapMode">
      <enum>QTextEdit::NoWrap</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="login_btn">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>100</y>
       <width>81</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Blippo Blk BT</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Login</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="inbody_info_1">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>130</y>
      <width>721</width>
      <height>761</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="inbody_img">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>13</y>
       <width>701</width>
       <height>741</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="guest_list">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>306</y>
      <width>241</width>
      <height>585</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QTableView" name="guest_table">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>221</width>
       <height>531</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="search_txt">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>547</y>
       <width>101</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>나눔바른고딕</family>
      </font>
     </property>
     <property name="cursor" stdset="0">
      <cursorShape>IBeamCursor</cursorShape>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhNone</set>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="tabChangesFocus">
      <bool>true</bool>
     </property>
     <property name="lineWrapMode">
      <enum>QTextEdit::NoWrap</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="search_btn">
     <property name="geometry">
      <rect>
       <x>176</x>
       <y>547</y>
       <width>25</width>
       <height>25</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="all_btn">
     <property name="geometry">
      <rect>
       <x>205</x>
       <y>547</y>
       <width>25</width>
       <height>25</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="left_btn">
    <property name="geometry">
     <rect>
      <x>943</x>
      <y>105</y>
      <width>25</width>
      <height>21</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background:url(:/Resource/img/left_icon.png)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="right_btn">
    <property name="geometry">
     <rect>
      <x>976</x>
      <y>105</y>
      <width>25</width>
      <height>21</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background:url(:/Resource/img/right_icon.png)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>파일</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>정보</string>
    </property>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
